### PR TITLE
[Backend] Introduce new API of Backend

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's introduce new API of Backend as commented yet.

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>>

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by jyoungyun** [comment]
I will wait for #486, #487 PR to be merged.
After merged these PRs, please uncomment your codes.

**Review by jyoungyun — APPROVED** [review]
LGTM

@YongseopKim intends to uncomment and use it whenever the user needs. It will be reflected as it is.

**Pull request merged**
